contracts/SpecificGroupStrategy.sol

```diff
@@ -334,6 +343,37 @@ contract SpecificGroupStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeab
         return (1, 1, 0, 0);
     }
 
+    /**
+     * @notice Used to withdraw CELO from the system from specific group strategy
+     * that account voted for previously. It is expected that strategy will be balanced.
+     * For balancing use `rebalance` function
+     * @param strategy The validator group that we want to withdraw from.
```

Comment:
NIT: ∆ `strategy` to `group`

As I understand, this whole contract is a strategy, and we vote for groups within this strategy.